plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version4/Refunds/class-wc-rest-refunds-v4-preview-tests.php

```
@@ -0,0 +1,1224 @@
+			}
+			// Stub always throws; the : array return type is never reached.
+			// phpcs:disable Squiz.Commenting.FunctionComment.InvalidNoReturn
+			/**
+			 * Always throws to exercise the controller's InvalidArgumentException catch arm.
+			 *
+			 * @param \WC_Order $order      Ignored.
+			 * @param array     $line_items Ignored.
+			 * @return array
+			 * @throws \InvalidArgumentException Always.
+			 */
+			public function build_refund_preview( \WC_Order $order, array $line_items ): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$line_items'. (undefined)

(UnusedFormalParameter)